opentasks/src/main/java/org/dmfs/tasks/EditTaskFragment.java

```diff
@@ -793,7 +793,9 @@ public void saveAndExit()
                 activity.finish();
                 if (isNewTask)
                 {
-                    activity.startActivity(new Intent("android.intent.action.VIEW", mTaskUri));
+                    activity.startActivity(
+                            new Intent(Intent.ACTION_VIEW, mTaskUri)
+                                    .putExtra(ViewTaskActivity.EXTRA_COLOR, TaskFieldAdapters.LIST_COLOR.get(mValues)));
```

Comment:
This won't work and causes the aforementioned crash because `mValues` doesn't contain the list color at this point. If the task is new, `mValues` has been created from scratch and not been loaded from the database, so it doesn't contain the list color yet.

Reply:
There is a field called `mListColor` which holds the color of the currently selected list.